db/migrate/20201113114332_add_uuid_to_course.rb

```diff
@@ -0,0 +1,13 @@
+class AddUuidToCourse < ActiveRecord::Migration[6.0]
+  def up
+    enable_extension "uuid-ossp" unless extension_enabled?("uuid-ossp")
+
+    add_column :course, :uuid, :uuid, default: "uuid_generate_v4()", null: false
+
+    add_index :course, :uuid, unique: true, name: "index_courses_unique_uuid"
+  end
+
+  def down
```

Comment:
This isn't an irreversible migration. Reversing is just dropping the column.

Reply:
split migrate as the extension is irreverible